plugins/logs/plugin.go

```diff
@@ -571,6 +586,12 @@ func (p *Plugin) Log(ctx context.Context, decision *server.Info) error {
 		inputAST:     decision.InputAST,
 	}
 
+	d, e := p.dropEvent(ctx, decision.Txn, &event)
+
+	if d || e != nil {
```

Comment:
Should we not do something about the error? Log it? Or even return it? What do we do for masking...? 🤔

Reply:
yes, maybe. I didn't dare to return it, because I wasn't sure what's really going on. I just worked from the code for masking decisions. Never used Go before.
Should I simply log it? And then drop the event?